app/styles/_shame.scss

```diff
@@ -13,7 +13,17 @@
 .data-table-pagination-left {
   @extend .au-c-help-text;
 }
-
+.helptext {
```

Comment:
Please use dashes to be consistent help-text link-external, active-status...

Reply:
Changed the names to be more consistent